vm/devices/tpm/src/ak_cert.rs

```diff
@@ -34,16 +34,15 @@ impl<T: 'static + GetAttestationReport> From<T> for ResolvedGetAttestationReport
     }
 }
 
+// Use the value defined by `tee_call::REPORT_DATA_SIZE`
+const REPORT_DATA_SIZE: usize = 64;
```

Comment:
it seems plausible that the TPM should simply pass a &[u8], and the backing implementation should report an error if the size isn't what it expects